libs/ui-components/src/components/Device/EditDeviceWizard/steps/DeviceTemplateStep.tsx

```diff
@@ -25,7 +25,7 @@ const DeviceTemplateStep = ({ isFleet }: { isFleet: boolean }) => {
 
   return (
     <Grid span={8}>
-      <Form>
+      <Form onSubmit={(ev) => ev.preventDefault()}>
```

Comment:
should we create a custom component with enter disabled ? Seems like it will get handy. Maybe even with eslint rule to remind us to use it

Reply:
Done, good idea!